### ISSUE TITLE
Scenes with Node as root spam two types of errors.

### ISSUE THREAD

**Issue by avencherus** [godotengine/godot#33244]
**Godot version:**
9fcd38fc7

**OS/device including version:**
Win10 64-bit

**Issue description:**
First error that keeps happening is when saving a scene that has a Node as its root, it will complain about not being able to generate and image.

![godot windows tools 64_DXtckZd9Ou](https://user-images.githubusercontent.com/13004169/68050830-ac05fb80-fcee-11e9-8c03-03f24a6fb36c.png)

Second error is when adding such a scene with a Node as root to a Node2D scene it will error about undo/redo position information, since it doesn't have a position.

![godot windows tools 64_UH0yJ92AYX](https://user-images.githubusercontent.com/13004169/68050892-cb9d2400-fcee-11e9-9e96-b962ae84e1d4.png)

**Minimal reproduction project:**
[Node to Node2D Errors.zip](https://github.com/godotengine/godot/files/3799340/Node.to.Node2D.Errors.zip)

**Comment by avencherus** [comment]
I don't often work with scenes of these root types, so I suspect these might be long standing, but it's not so clear to me what to search for, or if its been reported.

**Comment by KoBeWi** [comment]
I can't reproduce it in af4fd9de9 nor in 3.2 alpha3.

**Comment by Zylann** [comment]
In 3.2 alpha3 I can reproduce the second error when dragging the Node scene into another.
```
ERROR: _process_operation_list: Error calling method from signal 'set_position': 'Node::set_position': Method not found.
   At: core/undo_redo.cpp:296
```
The scene is properly added as child of the selected node.

**Comment by avencherus** [comment]
Having them in Alpha3 as well.  Does it actually vary between OS?

![errors](https://user-images.githubusercontent.com/13004169/68067187-02edee00-fd4c-11e9-8fef-a0c365e1adfc.jpg)

**Comment by aaronfranke** [comment]
Tested af4fd9de9c02052f3dc7330cd7cb6aa3f2107191, on both Ubuntu and Windows 10, can't reproduce either error.

**Comment by avencherus** [comment]
@aaronfranke Weird, is there anything configuration related that could be doing it?  Just simply saving the scene produces it every time for me in most versions I have, official and custom.

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/13004169/68068939-f5902e00-fd62-11e9-8bd2-ffe9e4334bc9.gif)

**Comment by aaronfranke** [comment]
@avencherus I don't know, do you have any special configurations? I haven't touched Godot's settings menu on Windows 10 as I do most of my development on Linux and just use Windows 10 to test things, so I have no special configurations on my end.

I suggest trying to reproduce this in a VM, if you can.

**Comment by KoBeWi** [comment]
@avencherus maybe upload you Godot settings here? There could be something that causes this.

**Comment by avencherus** [comment]
@KoBeWi I did a quick flush of certain app data files, and it has appeared to have gone away.  Upon restoring them and trying to uncover the offending file for comparison, it has stopped being reproducible.

So I suppose it was something in there triggering it, and the file had been re-arranged or cleaned up automatically by the editor when relaunching things.

Closing for the time being, it's likely an artifact of all different versions I alternate between.

Thanks everyone for taking a look into it.
